src/controllers/controller.doughnut.js

```diff
@@ -161,7 +161,7 @@ export default class DoughnutController extends DatasetController {
     const opts = me.options;
     const meta = me._cachedMeta;
     const circumference = me._getCircumference();
-    return reset && opts.animation.animateRotate ? 0 : this.chart.getDataVisibility(i) ? me.calculateCircumference(meta._parsed[i] * circumference / TAU) : 0;
+    return reset && opts.animation.animateRotate ? 0 : this.chart.getDataVisibility(i) ? (meta._parsed[i] === null ? 0 : me.calculateCircumference(meta._parsed[i]) * circumference / TAU) : 0;
```

Comment:
This looks like an error
`me.calculateCircumference(meta._parsed[i] * circumference / TAU)`
was changed to
`me.calculateCircumference(meta._parsed[i]) * circumference / TAU`

Reply:
I think this can just be reverted? `null * circumference / TAU` equals `0`

Reply:
wow, eagle eyes! good catch!  I've fixed this and also updated this section of code for readability to have a separate `if` check instead of multiple ternaries